lib/Runtime/Language/JavascriptConversion.h

```diff
@@ -12,10 +12,13 @@ namespace Js {
         static Var OrdinaryToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
         static Var MethodCallToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
         static Var ToPrimitive(Var aValue, JavascriptHint hint, ScriptContext * scriptContext);
-        static BOOL CanonicalNumericIndexString(Var aValue, double *indexValue, ScriptContext * scriptContext);
+        static BOOL CanonicalNumericIndexString(JavascriptString *aValue, double *indexValue, ScriptContext * scriptContext);
 
         static void ToPropertyKey(Var argument, ScriptContext* scriptContext, const PropertyRecord** propertyRecord);
 
+        static PropertyQueryFlags BooleanToPropertyQueryFlags(BOOL val) { return val ? Property_Found : Property_NotFound; }
+        static BOOL PropertyQueryFlagsToBoolean(PropertyQueryFlags val) { return val == Property_Found; }
```

Comment:
This is following line 27-30 below. Seems like "ToBoolean" => BOOL, and "ToBool" => bool.

---
In reply to: [106296732](https://github.com/Microsoft/ChakraCore/pull/2640#discussion_r106296732) [](ancestors = 106296732)